README.md

```diff
@@ -2,0 +1,17 @@
+# codacy-test
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/troubleshoot-codacy/codacy-test-fork-markdown-test/pullRequest?prid=7586344)
```suggestion

### Among Codacy’s features
```